pages/index.tsx

```diff
@@ -237,31 +189,25 @@ const Index: NextPage<{
                         onSettingsOpen={() => setIsSettingsOpen(true)}
                         onAgendaOpen={() => setIsAgendaOpen(true)}
                     />
-                    <WeekNavigator
```

Comment:
The move has altered the flex behavior. The schedule component should expand to fill the whole height.

![Screenshot from 2023-07-30 08-45-24](https://github.com/mathiazom/sit-rezervo-confgen/assets/24361490/82343f1f-ecaf-4a07-adc6-f64a57d8760e)

Reply:
Oh yeah, I did not forsee that. Luckily, I performed some funny circle composition, where WeekNavigator now is back in index.tsx. So I fixed this by moving the </Box> to where it originally was in https://github.com/mathiazom/sit-rezervo-confgen/pull/29/commits/90065a45e5f37a615f1eeee385a22f65db64e9d3 ✅